install

```diff
@@ -782,7 +782,7 @@ if [ -f /etc/airtime/airtime.conf ]; then
     
         set +e
         verbose "Stopping airtime services..."
-        systemInitCommand stop airtime_analyzer airtime-celery airtime-playout airtime-liquidsoap airtime-media-monitor
+        systemInitCommand stop libretime-analyzer airtime-celery airtime-playout airtime-liquidsoap airtime-media-monitor
```

Comment:
This will break upgrades where the new `libretime-analyzer` systemd unit has not been installed yet. Having all of these use the legacy aliases ensures that the first upgrade will work. After 1 release we can migrate them and require users to install alpha.9 before installing the next version

Reply:
Good catch! I rebased a fix.